netstandard/ref/System.Runtime.InteropServices.ComTypes.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace System.Runtime.InteropServices.ComTypes
 {
+    [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
```

Comment:
I'm not sure why all these types are hidden from intellisense in .NET Core.

@luqunl @jeffschwMSFT any ideas why these are all hidden?

Reply:
I also don't know why we add these EditorBrowsableAttribute. Is there any guideline to decide whether to add EditorBrowsableAttribute?

Reply:
I am not sure why these are being hidden. @terrajobst Is there a reason to hide them? Is this simply to avoid usage until we decide to support/not-support them?

Reply:
Also, why not use the actual value? (i.e. `EditorBrowsableState.Never`)

Reply:
>I am not sure why these are being hidden. @terrajobst Is there a reason to hide them? Is this simply to avoid usage until we decide to support/not-support them?

These are being marked hidden because the are currently hidden in .NET Core as we are adding APIs to the standard that were added in .NET Core. If they shouldn't be hidden in the standard then they shouldn't be hidden in .NET Core either. 

>Also, why not use the actual value? (i.e. EditorBrowsableState.Never)

That is a current tooling limitation we have when we generate the diffs we don't have support fro mapping the numerical values back to the Enum values.

Reply:
Oh. Sorry, I didn't see the repo wasn't coreclr 😊 Yes, then I get why it is happening. Thanks.

Also, boo for the enum issue...

Reply:
The question for you guys is should they be hidden? If we don't know why they are hidden then we shouldn't hide them here and we should unhide them in .NET Core. I'd also like for you guys to have a look at the APIs being included in here and see if there are any that should not be included in the standard.

Reply:
> Also, why not use the actual value? (i.e. `EditorBrowsableState.Never`)

Because it's a hard problem to solve. This source code is generated from IL which only has the final integer constant. It requires resolving the correct referenced assembly and reconstructing a possible combinations of enum fields. (I had to [solve this](https://github.com/jnm2/ApiContractGenerator/blob/5daf5a2ffe903400f56e730244772728b99be9bf/src/ApiContractGenerator/CSharpTextFormatter.cs#L1395-L1472) in my own project.)

Reply:
@luqunl @AaronRobinsonMSFT unless we think we should unhide these APIs in .NET Core, I'm going to take the with hidden.